### ISSUE TITLE
Stream: Chat and Org slides overlap each other

### ISSUE THREAD

**Issue by bkaestner** [FluxHarmonic/FluxHarmonic.github.io#1]
Didn't want to point it out during the stream, but I noticed that the chat and the org-mode slides overlay. You might want to move the chat section or add a background shadow. The topmost right corner is usually unused in many scenarios, so that might be suitable place?

Here's a screenshot:
![screenshot](https://user-images.githubusercontent.com/6301773/147570967-d8ca853c-526e-4f67-9d5d-cef524be985a.png)

**Comment by daviwil** [comment]
Yikes, thanks for pointing that out!  I'll try moving it higher and possibly find a way to put a shade behind the messages.

**Comment by daviwil** [comment]
I adjusted this a bit for the second stream, hopefully it was an improvement.  I still need to find a way to tweak the CSS of the chat box itself so that it has a translucent background to help it stand out a bit more.

Another weird thing I noticed on the SC stream Friday is that color of the chat sender names was a dark blue, different than what it is on the FH streams.  I'm not sure what controls that, but it made the names hard to read!
